jsk_arc2017_baxter/euslisp/pick-main.l

```diff
@@ -211,9 +220,10 @@
           (send *ri* :stop-grasp arm) ;; release object
           (send *ri* :spin-off-by-wrist arm :times 10)
           (send *ri* :wait-interpolation)
-          (send *ri* :angle-vector (send *baxter* arm :move-end-pos #f(0 0 400) :world)
+          (send *ri* :angle-vector-raw (send *baxter* arm :move-end-pos #f(0 0 400) :world)
```

Comment:
Why do you use `:angle-vector-raw`? You delete cardbard scene so that you can move robot in cardboard, don't you?